recogym/envs/abstract.py

```diff
@@ -285,7 +287,8 @@ def _store_bandit(action, reward):
                 data['z'].append('bandit')
                 data['v'].append(None)
                 data['a'].append(action['a'])
-                data['c'].append(reward)
+                data['c'].append(reward[0])
```

Comment:
I am not sure what is this.

Reply:
Now I see...
Maybe add a comment here saying that reward is actually (reward, ctr) or split it into two variables...

Reply:
A comment is added.